src/Operations.jl

```diff
@@ -1107,8 +1107,8 @@ function develop(ctx::Context, pkgs::Vector{PackageSpec}, new_git::Vector{UUID};
     update_manifest!(ctx, pkgs)
     new_apply = download_source(ctx, pkgs; readonly=true)
     download_artifacts(ctx, pkgs; platform=platform)
-    show_update(ctx)
     write_env(ctx.env) # write env before building
+    show_update(ctx)
```

Comment:
why after `write_env`? Wouldn't this make the `Updating` feedback misleading since we have already updated?

Reply:
I think it would be weird to have the update print successfully and then have the `write_env` fail. Then you would be shown that stuff updated but it actually didn't. So it seems better to me to only print the updating after it actually succeeded.

Also, this made it easier to check if a package was installed in the `show_update` function since then the updating of the project files is already done.